src/app/components/ConfirmOrCancel/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { MouseEventHandler } from "react";
```

Comment:
nit: This is called `ConfirmOrCancel` which sounds very general but it looks very specific. Maybe reflect this in the naming? Eventually we might need a general `ConfirmOrCancel`?

Reply:
Hmm unsure how to name it then.

Reply:
This is only being used in `ConfirmSignMessage` and comes with prepared texts matching that usage, yes? What about `ConfirmOrCancelSignMessage`?

Reply:
This is used in 8 screens.

Reply:
Ah, didn't see. Not aware of the context.   
I'm hestitating because it has this specific "Only connect with sites you trust.". Can it be named in relation to that?  
Could look into the screens and come up with something more matching tommrow.

Reply:
We might need to change the message or make it changable by a prop at some point. This sentence is used everywhere also when I want to send you a few sats for e.g.
So I'd say it's already a general component but we might need to improve on the sentence at some point.

Reply:
Ahhh, ok, well then... yolo